apps/ledger-live-mobile/src/components/FabActions/hooks/useAssetActions.tsx

```diff
@@ -123,9 +124,14 @@ export default function useAssetActions({
             ...(availableOnSwap
               ? [
                   {
-                    id: "swap",
```

Comment:
Why are we removing this it? Can we use the same structure for any item in the array?

Reply:
`apps/ledger-live-mobile/src/components/FabActions/index.tsx:127`

We want to add additional event properties like `swapVersion` etc, so we need to replace `id` with `event` in order to do that.